workload/htx_test.py

```diff
@@ -58,12 +58,21 @@ def install_latest_htx_rpm(self):
             htx_rpm for htx_rpm in matching_htx_versions
             if distro_pattern in htx_rpm]
         distro_specific_htx_versions.sort(reverse=True)
-        self.latest_htx_rpm = distro_specific_htx_versions[0]
-
-        if process.system('rpm -ivh --nodeps %s%s '
-                          '--force' % (self.rpm_link, self.latest_htx_rpm),
+        htx_latest_rpm = distro_specific_htx_versions[0]
+        self.latest_htx_rpm = htx_latest_rpm
+        tmp_dir = "/tmp/" + htx_latest_rpm
```

Comment:
the variable is slightly confusing, can you please rename it like 

`tmp_htx_rpm`